lib/ui/worker/cache.dart

```diff
@@ -660,6 +662,27 @@ class CacheEntry {
 
   /// Byte data of this [CacheEntry].
   final Uint8List? bytes;
+
+  /// MediaType type of this [CacheEntry].
+  late final Future<MediaType?> type = _resolveType();
+
+  Future<MediaType?> _resolveType() async {
```

Comment:
https://github.com/team113/messenger/blob/main/CONTRIBUTING.md#documentation-1

Reply:
/// Resolves [MediaType] of data in this [CacheEntry].